MISC/GetFrameTime.md

```diff
@@ -8,5 +8,23 @@ ns: MISC
 float GET_FRAME_TIME();
 ```
 
+Gets the high precision frame time of the last frame in seconds.
+
+Roughly equivalent to:
+```lua
```

Comment:
Examples go into their own section; see the [contribution guidelines](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md).